db/fixtures/lessons/javascript_lessons.rb

```diff
@@ -164,6 +164,63 @@ def javascript_lessons
       has_live_preview: true,
       identifier_uuid: '5212a020-85ed-4e43-84ec-6b583b57e8cb',
     },
+    'A Very Brief Intro to CS' => {
+      title: 'A Very Brief Intro to CS',
+      description: "Get a taste of what's coming up ahead and what the bigger world of CS looks like beyond the scope of this course.",
+      is_project: false,
+      github_path: '/javascript/computer_science/a_very_brief_intro_to_cs.md',
+      identifier_uuid: '460291a1-4d9f-4b33-a556-6fb25c541da9',
+    },
+    'Recursive Methods' => {
+      title: 'Recursive Methods',
+      description: 'Learn how making a function call itself can be helpful for making big problems into smaller problems',
+      is_project: false,
+      github_path: '/javascript/computer_science/recursive_methods.md',
+      identifier_uuid: 'b56213ff-8051-421e-8ca9-69b151e4d30e',
+    },
+    'Recursion' => {
+      title: 'Recursion',
+      description: 'Take your newfound knowledge and apply it to a couple of classic recursive problems',
+      is_project: true,
+      github_path: '/javascript/computer_science/project_recursion.md',
+      accepts_submission: true,
+      has_live_preview: false,
+      identifier_uuid: 'eaf37749-a971-4bd4-af79-6e166e55346a',
+    },
+    'Common Data Structures and Algorithms' => {
+      title: 'Common Data Structures and Algorithms',
+      description: 'Learn why we use different data structures to handle our data and some classic algorithms for searching through them to help solve problems',
+      is_project: false,
+      github_path: '/javascript/computer_science/common_data_structures_algorithms.md',
+      identifier_uuid: '8e973ed3-fde6-4c98-a9eb-c3c021616358',
+    },
+    'Linked Lists' => {
+      title: 'Linked Lists',
+      description: 'Build one of the most fundamental data structures.',
+      is_project: true,
+      github_path: '/javascript/computer_science/project_linked_lists.md',
+      accepts_submission: true,
+      has_live_preview: false,
+      identifier_uuid: 'd2c7044f-8a4a-4740-a455-e6467bcea25d',
+    },
+    'Binary Search Trees' => {
+      title: 'Binary Search Trees',
+      description: "In this project you'll get your hands dirty with one of the most common computer science problems, searching binary trees.",
+      is_project: true,
+      github_path: '/javascript/computer_science/project_binary_search_trees.md',
+      accepts_submission: true,
+      has_live_preview: false,
+      identifier_uuid: '6932753f-ca06-458e-af92-1871eaf5e211',
+    },
+    'Knights Travails' => {
+      title: 'Knights Travails',
+      description: 'Lets build Knights Travails',
```

Comment:
Nit - I think this could be a bit more descriptive. Is this the same as the Ruby one?

Not a blocker

Reply:
Yeah it was just ripped from the ruby course. This can be addressed in a separate PR if you want.

Reply:
@CouchofTomato Having a look through, I can't see anywhere these are actually used (outside of when they're seeded). 

If that's the case I wouldn't worry, not sure if they're actually needed for anything :shrug: 
![image](https://user-images.githubusercontent.com/58506115/182824538-ed055f14-f859-4583-8a87-8e4eefaf44b7.png)